### PR TITLE
fix(perf): add link to transaction summary on performance change explorer

### PR DESCRIPTION
**Pull request by nikkikapadia**
Addressing the feedback that @udameli and @narsaynorath gave about having a link to the transaction summary from the performance change explorer. This adds a link at the bottom of the performance change explorer which directs users to the transaction summary with the trends graph displayed.

Screenshot 2023-07-12 at 12 11 35 PM

EDIT: linked to transaction summary through open icon in header instead
image

### CODE REVIEW

File: static/app/views/performance/trends/changeExplorer.tsx [hunk 1]
```diff
@@ -177,12 +214,47 @@ function Header(props: HeaderProps) {
         <ChangeType regression={regression}>
           {regression ? t('Ongoing Regression') : t('Ongoing Improvement')}
         </ChangeType>
-        <TransactionName>{transaction.transaction}</TransactionName>
+        <div style={{display: 'flex', alignItems: 'center', marginBottom: space(3)}}>
+          <TransactionName style={{width: 'fit-content'}}>
```
Comment (narsaynorath) [inline static/app/views/performance/trends/changeExplorer.tsx:218]: Can we just stick `width` into the styled component as well?

File: static/app/views/performance/trends/changeExplorer.tsx [hunk 2]
```diff
@@ -177,12 +214,47 @@ function Header(props: HeaderProps) {
         <ChangeType regression={regression}>
           {regression ? t('Ongoing Regression') : t('Ongoing Improvement')}
         </ChangeType>
-        <TransactionName>{transaction.transaction}</TransactionName>
+        <div style={{display: 'flex', alignItems: 'center', marginBottom: space(3)}}>
```
Comment (narsaynorath) [inline static/app/views/performance/trends/changeExplorer.tsx:217]: We should make this a styled component as well, we can call it `TransactionNameWrapper` just to tidy this part up

File: static/app/views/performance/trends/changeExplorer.tsx [hunk 3]
```diff
@@ -177,12 +214,47 @@ function Header(props: HeaderProps) {
...
+          <TransactionName style={{width: 'fit-content'}}>
+            {transaction.transaction}
+          </TransactionName>
+          <Button
+            style={{padding: space(0), height: 'min-content', minHeight: '0px'}}
```
Comment (narsaynorath) [inline static/app/views/performance/trends/changeExplorer.tsx:222]: You can also make this a styled component and call it `ViewTransactionButton`

File: static/app/views/performance/trends/changeExplorer.tsx [hunk 4]
```diff
@@ -164,9 +185,25 @@ function InfoItem({label, value}: {label: string; value: string}) {
...
+    trendChangeType,
+    trendView,
+    projects,
+    organization,
+    trendFunction,
+    trendParameter,
+  } = props;
 
   const regression = trendChangeType === TrendChangeType.REGRESSION;
+  const next = getTransactionSummaryLink(
```
Comment (narsaynorath) [inline static/app/views/performance/trends/changeExplorer.tsx:199]:
```suggestion
  const transactionSummaryLink = getTransactionSummaryLink(
```

Doesn't hurt to be explicit here, `next` didn't really tell me much about what the variable was for

**Review by narsaynorath — APPROVED** [review]
Small comments, but lgtm. I ran it locally and it's great!

File: static/app/views/performance/trends/changeExplorer.tsx (hunk 4 above)
Comment (nikkikapadia) [reply static/app/views/performance/trends/changeExplorer.tsx:199]: AHH truly embarrassing I forgot to change it before I committed 🥲

**Pull request merged**